src/module.c

```diff
@@ -597,7 +597,7 @@ int ReplySeriesRange(RedisModuleCtx *ctx,
     }
 
     RedisModule_ReplyWithArray(ctx, REDISMODULE_POSTPONED_ARRAY_LEN);
-    if (aggObject == TS_AGG_NONE) {
+    if (aggObject == NULL) {
```

Comment:
@ashtul @rafie how is this related to the PR?

Reply:
It isn't.
It's a small mistake since `aggObject` is a pointer but `TS_AGG_NONE` value is `0` so we didn't get a warning.